internal/agent/device/device.go

```diff
@@ -236,118 +289,100 @@ func (a *Agent) beforeUpdateApplications(ctx context.Context, _, desired *v1alph
 				return true, nil
 			})
 			if err != nil {
-				return fmt.Errorf("failed to pull image %q: %w", providerImage, err)
+				return fmt.Errorf("creating enrollment request: %w", err)
```

Comment:
Nit: Possible rebase error? The previous message was better.

Reply:
I swear I fixed this haha

Reply:
fixed